paddle/fluid/pybind/eager_utils.cc

```diff
@@ -305,7 +305,7 @@ std::vector<int> CastPyArg2VectorOfInt(PyObject* obj, size_t arg_pos) {
     Py_ssize_t len = PyList_Size(obj);
     PyObject* item = nullptr;
     for (Py_ssize_t i = 0; i < len; i++) {
-      item = PyList_GetItem(obj, i);
+      item = PyList_GET_ITEM(obj, i);
```

Comment:
https://docs.python.org/3/c-api/list.html?highlight=pylist_getitem#c.PyList_GetItem
<img width="916" alt="image" src="https://user-images.githubusercontent.com/31559413/188305585-0eb9c24e-2033-4315-8d9d-3983ebb6507c.png">
<img width="808" alt="image" src="https://user-images.githubusercontent.com/31559413/188305665-d42f1064-c4bb-4c2d-ac5d-8cbbb7b1b0fa.png">
<img width="960" alt="image" src="https://user-images.githubusercontent.com/31559413/188305825-811c941c-7a81-4a4c-997a-9550563b59cf.png">

In this place, actually, you don't have to do any error checking on "obj" and "i",  so PyList_GetItem  can be changed to  PyList_GET_ITEM